jwst/tweakreg/tweakreg_step.py

```diff
@@ -200,6 +207,61 @@ def process(self, input):
             else:
                 raise e
 
+        if self.align_to_gaia:
+            try:
+                # Get catalog of GAIA sources for the field
+                #
+                # NOTE:  If desired, the pipeline can write out the reference catalog
+                #        as a separate product with a name based on whatever convention
+                #        is determined by the JWST Cal Working Group
+                if self.output_gaia:
+                    output_name = 'fit_{}_ref.ecsv'.format(self.gaia_catalog.lower())
+                else:
+                    output_name = None
+                ref_cat = amutils.create_astrometric_catalog(self.input_models,
+                                                             self.gaia_catalog,
+                                                             output=output_name)
+
+                # Check that there are enough GAIA sources for a reliable/valid fit
+                num_ref = len(ref_cat)
+                if num_ref < self.minobj:
+                    msg = "Not enough GAIA sources for a fit: {}\n".format(num_ref)
+                    msg += "Skipping alignment to {} astrometric catalog!\n".format(self.gaia_catalog)
+                    raise ValueError(msg)
```

Comment:
Is this raising of an error going to cause the step and pipeline to halt, or is the error trapped up above, so that the rest of the pipeline can proceed without any modifications to the WCS? What we prefer is the latter (in which case this might need to be changed to just a warning message, rather than raising an error).

Reply:
This Exception gets trapped at line 259 (or thereabouts) so that the only result is a warning being reported in the log and the rest of the processing proceeds normally with no changes to the WCS.

Reply:
Fixed in 1df551e (subject to continued testing...).